Source/monster.cpp

```diff
@@ -662,15 +662,15 @@ void LoadDiabMonsts()
 	}
 }
 
-void DeleteMonster(int i)
+void DeleteMonster(int monsterId)

```

Comment:
This function doesn't take a monster ID, it's an index into the list of active monster IDs stored in ActiveMonsters. `i` is perfectly fine here, or `size_t activeIndex` if you want to be verbose.

Reply:
Thanks! I went with `activeIndex` as I like verbose 😛